### ISSUE TITLE
Consistency around flagging errors in Pull Requests

### ISSUE THREAD

**Issue by mattheu** [humanmade/linter-bot#78]
If I open a pull request, linter bot will only block merging if there are errors in code changed as part of the Pull Request. This is good 👍 

But if I go to the 'checks' tab, I see all errors on the project, not just those affecting the PR's.

- As a project lead, I would NOT want a developer to fix these unrelated issues as part of this PR.
- As a developer it can hard to see what errors I need to fix as part of this Pull Request. Especially for large PR's where the 'changes' view is hard to scan.

Obviously - having no other errors on the project would be ideal - but the reality is that sometimes they sneak in. I think we need to be consistent. If they don't block the PR, don't show them - or at least indicate this.

Regarding surfacing errors on the whole project? I agree this would be useful, but I don't think an unrelated PR is the right place.

**Comment by roborourke** [comment]
Yeah, errors for the whole project would ideally be surfaced locally just using phpcs or similar
